src/main/java/mod/crend/autohud/component/ValueComponentState.java

```diff
@@ -26,7 +27,7 @@ public void update() {
     }
 
     protected boolean doReveal(T newValue) {
-        return !component.config.active() || newValue != oldValue;
+        return !component.config.active() || !Objects.equals(newValue, oldValue);
```

Comment:
Fixed it for you :wink:

Reply:
Ah, I didn't see this. I chose the same solution. :)